features/welcome.feature

```diff
@@ -3,3 +3,7 @@ Feature: Welcome screen
   Scenario: Viewing the welcome screen
     When I visit the home page
     Then I should see a warm welcome message
+    Then I should see "Receipt Yourself"
+    Then I should see "Register"
+    Then I should see "Log In"
+
```

Comment:
Maybe add a scenario for:

1. Registering a new user
2. Logging in of a existing user?